src/sentry/event_manager.py

```diff
@@ -2049,6 +2055,34 @@ def _get_severity_metadata_for_group(event: Event) -> Mapping[str, Any]:
     return {}
 
 
+def _get_priority_for_group(severity: Mapping[str, Any], kwargs: Mapping[str, Any]) -> int:
+    """
+    Returns priority for an event based on severity score and log level.
+    """
+    level = kwargs.get("level", None)
+    severity_score = severity.get("severity", None)
+
+    if level in [logging.INFO, logging.DEBUG]:
+        return PriorityLevel.LOW
+
+    elif level == logging.FATAL:
+        return PriorityLevel.HIGH
+
+    elif level == logging.WARNING:
+        if not severity_score or severity_score < 0.1:
```

Comment:
Feels like maybe the 0.1 should be a constant, `HIGH_PRIORITY_THRESHOLD` or something?